tdm-admin/src/app/util/utils.ts

```diff
@@ -0,0 +1,43 @@
+import type { Page } from '~/app/util/type';
+
+const formatBytes = (bytes: number, decimals = 2) => {
+    if (!bytes || bytes === 0) {
+        return '0 Bytes';
+    }
+
+    const k = 1024;
+    const dm = decimals < 0 ? 0 : decimals;
+    const sizes = ['Bytes', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB', 'ZiB', 'YiB'];
+
+    const i = Math.floor(Math.log(bytes) / Math.log(k));
+
+    return `${parseFloat((bytes / Math.pow(k, i)).toFixed(dm))} ${sizes[i]}`;
+};
+
+export const getPageTitle = (page: Page) => {
+    switch (page) {
+        case 'home':
+            return 'Tableau de bord';
+        case 'processing':
+            return 'Enrichissement';
+        case 'database':
+            return 'Base de donnée';
+        case 'file':
+            return 'Fichier';
+        case 'log':
+            return 'Journal';
+        case 'setting':
+            return 'Paramètre';
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** 

Je suppose qu'il y a potentiellement plusieurs enrichissements.
Et plusieurs paramètres. Et plusieurs fichiers.
Je pense que pour les journaux ce serait pertinent aussi.
‏
```suggestion
            return 'Tableau de bord';
        case 'processing':
            return 'Enrichissements';
        case 'database':
            return 'Base de données';
        case 'file':
            return 'Fichiers';
        case 'log':
            return 'Journaux';
        case 'setting':
            return 'Paramètres';
```